flow360/component/simulation/services.py

```diff
@@ -96,6 +102,13 @@ def get_default_params(unit_system_name, length_unit) -> SimulationParams:
             reference_geometry=ReferenceGeometry(
                 area=1, moment_center=(0, 0, 0), moment_length=(1, 1, 1)
             ),
+            meshing=MeshingParams(
+                refinements=[
+                    SurfaceRefinement(name="Surface refinement", max_edge_length=1),
```

Comment:
The `name` here is for display right? Then how about have it as `Global surface refinement setting`?

Reply:
max_edge_length is no longer a required field. After this PR: https://github.com/flexcompute/Flow360/pull/352